src/main/scala/com/crealytics/spark/excel/ExcelRelation.scala

```diff
@@ -108,6 +109,20 @@ case class ExcelRelation(
     }
   }
 
+  private def getCellValue(cell: Cell): String = {
```

Comment:
There's similar code [here](https://github.com/crealytics/spark-excel/blob/81ac73eb00b92f1ca3fc5c863ba7ce45b9f7306c/src/main/scala/com/crealytics/spark/excel/DataColumn.scala#L26-L90). Could this be reused?